lib/Backend/LowerMDShared.cpp

```diff
@@ -8684,6 +8684,18 @@ void LowererMD::GenerateFastInlineBuiltInCall(IR::Instr* instr, IR::JnHelperMeth
         Assert(helperMethod == (IR::JnHelperMethod)0);
         return GenerateFastInlineBuiltInMathAbs(instr);
 
+    case Js::OpCode::InlineMathPow:
+#ifdef _M_IX86
+        if (!instr->GetSrc2()->IsFloat())
```

Comment:
The function called by Math.pow(double, double) has naked calling convention on x86 so it need to fallthough to previous path.

---
In reply to: [57631625](https://github.com/Microsoft/ChakraCore/pull/644#discussion_r57631625) [](ancestors = 57631625)